src/react-components/src/components/03-organisms/EditionsList/EditionsList.tsx

```diff
@@ -1,4 +1,6 @@
 // OL-24 Editions List
+/** DEPRECATION WARNING, please build with <UnorderedList> component */
```

Comment:
Are these deprecation warnings just to make the SFR integration easier? (This is just ignorance on my part. I assume there's a myriad of ways to communicate upcoming deprecations to users of your library?)

Reply:
In the future we'd announce deprecations in patch releases and remove them in major releases.  This is going to be a "major" release so it's mostly for me to remember to delete these files.